[BUG] Xcparse not installing with Xcode 13.0 Beta 4

**Describe the bug**
Xcparse is failing to install when Xcode 13.0 Beta 4 is installed.
```
/private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-package-manager/Sources/Basic/Await.swift:18:28: error: value of type '(@escaping (Result<T, ErrorType>) -> Void) -> Void' has no member 'dematerialize'
    return try await(body).dematerialize()
                    ~~~~~~ ^~~~~~~~~~~~~
make: *** [build] Error 1
```

**Desktop (please complete the following information):**
 - OS: MacOS1 1.5.1
 - xcparse Version 2.1.2
 - Swift version 5.5


**Crash Report**
```
==> Tapping chargepoint/xcparse
Cloning into '/usr/local/Homebrew/Library/Taps/chargepoint/homebrew-xcparse'...
remote: Enumerating objects: 77, done.        
remote: Total 77 (delta 0), reused 0 (delta 0), pack-reused 77        
Receiving objects: 100% (77/77), 8.59 KiB | 2.15 MiB/s, done.
Resolving deltas: 100% (33/33), done.
Tapped 1 formula (15 files, 17.9KB).
==> Installing xcparse from chargepoint/xcparse
==> Cloning https://github.com/ChargePoint/xcparse.git
Cloning into '/Users/distiller/Library/Caches/Homebrew/xcparse--git'...
==> Checking out tag 2.1.2
HEAD is now at 61bfe78 Arrays of XCResultValueTypes do not parse correctly (#52)
==> make install prefix=/usr/local/Cellar/xcparse/2.1.2
Last 15 lines from /Users/distiller/Library/Logs/Homebrew/xcparse/01.make:
Computing version for https://github.com/apple/swift-package-manager.git
Computed https://github.com/apple/swift-package-manager.git at 0.5.0 (6.46s)
Computing version for https://github.com/apple/swift-llbuild.git
Computed https://github.com/apple/swift-llbuild.git at 0.2.0 (0.46s)
Creating working copy for https://github.com/apple/swift-package-manager.git
Working copy of https://github.com/apple/swift-package-manager.git resolved at 0.5.0
Creating working copy for https://github.com/apple/swift-llbuild.git
Working copy of https://github.com/apple/swift-llbuild.git resolved at 0.2.0
'llbuild' /private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild: warning: ignoring broken symlink /private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild/lib/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm
'llbuild' /private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild: warning: Invalid Exclude '/private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild/products/llbuildSwift/llbuild.swift': File not found.
[1/2] Compiling clibc libc.c
remark: Incremental compilation has been disabled: it is not compatible with whole module optimizationremark: Incremental compilation has been disabled: it is not compatible with whole module optimization/private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-package-manager/Sources/Basic/Await.swift:18:28: error: value of type '(@escaping (Result<T, ErrorType>) -> Void) -> Void' has no member 'dematerialize'
    return try await(body).dematerialize()
                    ~~~~~~ ^~~~~~~~~~~~~
make: *** [build] Error 1
```

**To Reproduce**
Run `brew install chargepoint/xcparse/xcparse` to install xcparse with Xcode 13.0 Beta 4 installed.

**Expected behavior**
Xcparse should install successfully

--- a/Package.resolved
+++ b/Package.resolved
@@ -2,21 +2,12 @@
   "object": {
     "pins": [
       {
-        "package": "llbuild",
-        "repositoryURL": "https://github.com/apple/swift-llbuild.git",
+        "package": "swift-tools-support-core",
+        "repositoryURL": "https://github.com/apple/swift-tools-support-core.git",
         "state": {
           "branch": null,
-          "revision": "f73b84bc1525998e5e267f9d830c1411487ac65e",
-          "version": "0.2.0"
-        }
-      },
-      {
-        "package": "SwiftPM",
-        "repositoryURL": "https://github.com/apple/swift-package-manager.git",
-        "state": {
-          "branch": null,
-          "revision": "9abcc2260438177cecd7cf5185b144d13e74122b",
-          "version": "0.5.0"
+          "revision": "3b6b97d612b56e25d80d0807f5bc38ea08b7bdf3",
+          "version": "0.2.3"
         }
       }
     ]

--- a/Package.swift
+++ b/Package.swift
@@ -18,17 +18,17 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(url: "https://github.com/apple/swift-package-manager.git", .exact("0.5.0")),
+        .package(url: "https://github.com/apple/swift-tools-support-core.git", .exact("0.2.3")),
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages which this package depends on.
         .target(
             name: "xcparse",
-            dependencies: [ "XCParseCore", "SPMUtility" ]),
+            dependencies: [ "XCParseCore", "SwiftToolsSupport" ]),
         .target(
             name: "XCParseCore",
-            dependencies: [ "SPMUtility" ]),
+            dependencies: [ "SwiftToolsSupport" ]),
         .testTarget(
             name: "xcparseTests",
             dependencies: ["xcparse"]),

--- a/Sources/XCParseCore/Console.swift
+++ b/Sources/XCParseCore/Console.swift
@@ -6,8 +6,8 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
+import TSCBasic
 
 public enum OutputType {
   case error
@@ -42,7 +42,7 @@ open class Console {
     @discardableResult public func shellCommand(_ command: [String]) -> String {
         self.writeMessage("Command: \(command.joined(separator: " "))\n", to: .verbose)
 
-        let process = Basic.Process(arguments: command)
+        let process = TSCBasic.Process(arguments: command)
         do {
             try process.launch()
             let result = try process.waitUntilExit()

--- a/Sources/XCParseCore/Version+XCPTooling.swift
+++ b/Sources/XCParseCore/Version+XCPTooling.swift
@@ -6,7 +6,7 @@
 //
 
 import Foundation
-import SPMUtility
+import TSCUtility
 
 public extension Version {
     static func xcresulttoolCompatibleWithUnicodeExportPath() -> Version {

--- a/Sources/XCParseCore/XCResultToolCommand.swift
+++ b/Sources/XCParseCore/XCResultToolCommand.swift
@@ -6,14 +6,14 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCBasic
+import TSCUtility
 
 let xcresultToolArguments = ["xcrun", "xcresulttool"]
 
 open class XCResultToolCommand {
-    let process: Basic.Process
+    let process: TSCBasic.Process
 
     let xcresult: XCResult
     var console: Console {
@@ -22,12 +22,12 @@ open class XCResultToolCommand {
         }
     }
 
-    public init(withXCResult xcresult: XCResult, process: Basic.Process = Basic.Process(arguments: ["xcrun", "xcresulttool", "-h"])) {
+    public init(withXCResult xcresult: XCResult, process: TSCBasic.Process = TSCBasic.Process(arguments: ["xcrun", "xcresulttool", "-h"])) {
         self.xcresult = xcresult
         self.process = process
     }
     
-    @discardableResult public func run() -> Basic.ProcessResult? {
+    @discardableResult public func run() -> TSCBasic.ProcessResult? {
         do {
             self.console.writeMessage("Command: \(process.arguments.joined(separator: " "))\n", to: .verbose)
 
@@ -76,7 +76,7 @@ open class XCResultToolCommand {
                                             "--id", self.id,
                                             "--output-path", self.outputPath])
 
-            let process = Basic.Process(arguments: processArgs)
+            let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
 
@@ -97,7 +97,7 @@ open class XCResultToolCommand {
                                             "--id", self.id,
                                             "--output-path", self.outputPath])
 
-            let process = Basic.Process(arguments: processArgs)
+            let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
     }
@@ -128,7 +128,7 @@ open class XCResultToolCommand {
                 processArgs.append(contentsOf: ["--output-path", self.outputPath])
             }
 
-            let process = Basic.Process(arguments: processArgs)
+            let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
     }
@@ -151,7 +151,7 @@ open class XCResultToolCommand {
                 processArgs.append(contentsOf: ["--version", "\(version)"])
             }
 
-            let process = Basic.Process(arguments: processArgs)
+            let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
     }
@@ -163,7 +163,7 @@ open class XCResultToolCommand {
             processArgs.append(contentsOf: ["metadata", "get",
                                             "--path", xcresult.path])
 
-            let process = Basic.Process(arguments: processArgs)
+            let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
     }
@@ -175,7 +175,7 @@ open class XCResultToolCommand {
             processArgs.append(contentsOf: ["version"])
 
             let xcresult = XCResult(path: "")
-            let process = Basic.Process(arguments: processArgs)
+            let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
     }

--- a/Sources/xcparse/AttachmentsCommand.swift
+++ b/Sources/xcparse/AttachmentsCommand.swift
@@ -6,9 +6,9 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCBasic
+import TSCUtility
 
 struct AttachmentsCommand: Command {
     let command = "attachments"
@@ -59,7 +59,7 @@ struct AttachmentsCommand: Command {
         }
         let xcresultPath = xcresultPathArgument.path
 
-        var outputPath: AbsolutePath
+        var outputPath: TSCBasic.AbsolutePath
         if let outputPathArgument = arguments.get(self.outputPath) {
             outputPath = outputPathArgument.path
         } else if let workingDirectory = localFileSystem.currentWorkingDirectory {

--- a/Sources/xcparse/CodeCoverageCommand.swift
+++ b/Sources/xcparse/CodeCoverageCommand.swift
@@ -6,9 +6,9 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCBasic
+import TSCUtility
 
 struct CodeCoverageCommand: Command {
     let command = "codecov"
@@ -35,7 +35,7 @@ struct CodeCoverageCommand: Command {
         }
         let xcresultPath = xcresultPathArgument.path
 
-        var outputPath: AbsolutePath
+        var outputPath: TSCBasic.AbsolutePath
         if let outputPathArgument = arguments.get(self.outputPath) {
             outputPath = outputPathArgument.path
         } else if let workingDirectory = localFileSystem.currentWorkingDirectory {

--- a/Sources/xcparse/Command.swift
+++ b/Sources/xcparse/Command.swift
@@ -6,9 +6,8 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCUtility
 
 // This is cribbed form a great blog post on ArgumentParser
 // https://www.enekoalonso.com/articles/handling-commands-with-swift-package-manager

--- a/Sources/xcparse/CommandRegistry.swift
+++ b/Sources/xcparse/CommandRegistry.swift
@@ -6,9 +6,9 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCBasic
+import TSCUtility
 
 // This is cribbed form a great blog post on ArgumentParser
 // https://www.enekoalonso.com/articles/handling-commands-with-swift-package-manager
@@ -113,7 +113,7 @@ struct CommandRegistry {
         // We've determined it isn't a legacy command, so use new parsing
         guard let subparser = arguments.subparser(parser),
             let command = commands.first(where: { $0.command == subparser }) else {
-            parser.printUsage(on: stdoutStream)
+            parser.printUsage(on: TSCBasic.stdoutStream)
             return
         }
         try command.run(with: arguments)

--- a/Sources/xcparse/LogsCommand.swift
+++ b/Sources/xcparse/LogsCommand.swift
@@ -6,9 +6,9 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCBasic
+import TSCUtility
 
 struct LogsCommand: Command {
     let command = "logs"
@@ -35,7 +35,7 @@ struct LogsCommand: Command {
         }
         let xcresultPath = xcresultPathArgument.path
 
-        var outputPath: AbsolutePath
+        var outputPath: TSCBasic.AbsolutePath
         if let outputPathArgument = arguments.get(self.outputPath) {
             outputPath = outputPathArgument.path
         } else if let workingDirectory = localFileSystem.currentWorkingDirectory {

--- a/Sources/xcparse/ScreenshotsCommand.swift
+++ b/Sources/xcparse/ScreenshotsCommand.swift
@@ -6,9 +6,9 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCBasic
+import TSCUtility
 
 struct ScreenshotsCommand: Command {
     let command = "screenshots"
@@ -61,7 +61,7 @@ struct ScreenshotsCommand: Command {
         }
         let xcresultPath = xcresultPathArgument.path
 
-        var outputPath: AbsolutePath
+        var outputPath: TSCBasic.AbsolutePath
         if let outputPathArgument = arguments.get(self.outputPath) {
             outputPath = outputPathArgument.path
         } else if let workingDirectory = localFileSystem.currentWorkingDirectory {

--- a/Sources/xcparse/VersionCommand.swift
+++ b/Sources/xcparse/VersionCommand.swift
@@ -6,9 +6,8 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCUtility
 import XCParseCore
 
 struct VersionCommand: Command {

--- a/Sources/xcparse/XCPParser.swift
+++ b/Sources/xcparse/XCPParser.swift
@@ -6,9 +6,9 @@
 //  Copyright © 2019 ChargePoint, Inc. All rights reserved.
 //
 
-import Basic
 import Foundation
-import SPMUtility
+import TSCBasic
+import TSCUtility
 import XCParseCore
 
 let xcparseCurrentVersion = Version(2, 1, 0)
@@ -272,7 +272,7 @@ class XCPParser {
         }
 
         let header = (displayName != "") ? "Exporting \"\(displayName)\" Attachments" : "Exporting Attachments"
-        let progressBar = PercentProgressAnimation(stream: stdoutStream, header: header)
+        let progressBar = PercentProgressAnimation(stream: TSCBasic.stdoutStream, header: header)
         progressBar.update(step: 0, total: attachments.count, text: "")
 
         for (index, attachment) in attachments.enumerated() {

